docs/standard/serialization/system-text-json-migrate-from-newtonsoft-how-to.md

```diff
@@ -381,7 +381,12 @@ To support a dictionary with an integer or some other type as the key in .NET Co
 <xref:System.Text.Json> doesn't provide built-in support for the following types:
 
 * <xref:System.Data.DataTable> and related types
-* F# types, such as [discriminated unions](../../fsharp/language-reference/discriminated-unions.md), [record types](../../fsharp/language-reference/records.md), and [anonymous record types](../../fsharp/language-reference/anonymous-records.md).
+::: zone pivot="dotnet-5-0"
+* F# types, such as [discriminated unions](../../fsharp/language-reference/discriminated-unions.md). [Record types](../../fsharp/language-reference/records.md) and [anonymous record types](../../fsharp/language-reference/anonymous-records.md) are treated as immutable POCOs and thus are supported. Use [FSharp.SystemTextJson](https://github.com/Tarmil/FSharp.SystemTextJson) for a community-implemented custom converter to support other F# types.
```

Comment:
In this PR, we should only mention that there is support for F# records, and remove the reference to `FSharp.SystemTextJson`. We shouldn't endorse a particular library in this manner.

Reply:
Or use the term "a library"?

Reply:
System.Text.Json maintainers have not used or tested this library. We should not endorse it in public, official documentation.